instance.tf

```diff
@@ -24,7 +24,7 @@ resource "aws_instance" "web" {
   user_data = file("apache-script.sh")
 
   tags = {
-    Name = "Hello-DevOps"
+    Name = "Hello-DevOps_World"
```

Comment:
This change is not related to the remote state feature, please remove it from this pull request